Neo4jClient.Shared/Attributes/Neo4jIgnoreAttribute.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Neo4jClient
+
+{
+    public class Neo4jIgnoreAttribute : Attribute
```

Comment:
Could you add:
`[AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]`

To this, just to restrict the usage

Reply:
Done.